config/geonotebook.ini

```diff
@@ -1,7 +1,16 @@
 [default]
-vis_server = geoserver
+vis_server = ktile
 
 [geoserver]
 username = admin
 password = geoserver
 url = http://127.0.0.1:8080/geoserver
+
+[ktile]
+url = http://127.0.0.1:8888/ktile
+default_cache = ktile_default_cache
+
+[ktile_default_cache]
+name = Test
+path = /tmp/stache
+umask = 0000
```

Comment:
Should the defaults for the ansible geonotebook role be changed as a result of this?